Userland/Services/NotificationServer/NotificationWindow.cpp

```diff
@@ -46,7 +46,7 @@ static void update_notification_window_locations(const Gfx::IntRect& screen_rect
         auto& window = window_entry.value;
         Gfx::IntPoint new_window_location;
         if (last_window_rect.is_null())
-            new_window_location = screen_rect.top_right().translated(-window->rect().width() - 24, 26);
+            new_window_location = screen_rect.top_right().translated(-window->rect().width() - 24, 7);
```

Comment:
Maybe these magic numbers could be documented with a constexpr variable somewhere?